plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -205,24 +211,29 @@ def copy_rpms(units, source_repo, dest_repo, import_conduit, config, copy_deps,
     :type  import_conduit:  pulp.plugins.conduits.unit_import.ImportUnitConduit
     :param config:          configuration instance passed to the importer of the destination repo
     :type  config:          pulp.plugins.config.PluginCallConfiguration
-    :param copy_deps:       if True, copies dependencies as specified in "Requires"
-                            lines in the RPM metadata. Matches against NEVRAs
-                            and Provides declarations that are found in the
-                            source repository. Silently skips any dependencies
-                            that cannot be resolved within the source repo.
-    :param solver:          an object that can be used for dependency solving.
-                            this is useful so that data can be cached in the
-                            depsolving object and re-used by each iteration of
-                            this method.
-    :type  solver:          pulp_rpm.plugins.importers.yum.depsolve.Solver
+    :param solver:          if not None, it is used to resolve dependencies as specified in
+                            "Requires" lines in the RPM metadata.
 
+                            # FIXME: not really true anymore:
```

Comment:
could you just remove the fixme since it's a not a problem and just leaving the note of silent skip of unresolvable deps.